nemo/lightning/io/connector.py

```diff
@@ -212,6 +212,10 @@ def local_path(self, base_path: Optional[Path] = None) -> Path:
 
             _base = Path(NEMO_MODELS_CACHE)
 
+        # If the use supplied `hf:///path/to/downloaded/my-model/`
+        # then extract the last dir-name (i.e. my-model) and append it to _base
+        if str(self).startswith('/'):
```

Comment:
Why is it appended to base? `/` at the beginning would indicate an absolute path right?

Reply:
the goal of the `local_path` is to return the path where the converted checkpoint will be written, so in this case it will be NEMO_MODELS_CACHE / my-model